ci

```diff
@@ -448,6 +448,25 @@ case "$1" in
     refresh_fstar_hints
     ;;
 
+  fstar-docs-nightly)
+    # Create the docs associated with FStar
+    if [ ! -d ulib ]; then
+      echo "I don't seem to be in the right directory, bailing"
+      exit 1
+    fi
+
+    # First - get fstar built
+    git clean -ffdx
+    git reset --hard HEAD
+    git pull origin master
+    make -C src/ocaml-output clean
+    make -C src/ocaml-output -j 6
+
+    # Second - run fstar with the --doc flag
+    chmod +x .ci/fsdoc.sh
```

Comment:
I ran into case where it wasn't executable so I was just making sure it would be so wouldn't have issues. Will remove the chmod and if it isn't working will fix in git repo